### ISSUE TITLE
Increase conversion with original "Get is on GooglePlay" button

### ISSUE THREAD

**Issue by Apollo144** [megacool/sdk#32]
Hi guys!
Why is there pink button on the landing page? Did you try split testing and pink button performed better?
https://imgurhd.ru/i/3zxb_orig.png
Can we change it to original 'Get it on GooglePlay' button?
https://imgurhd.ru/i/3zxc_orig.png

In many countries people are afraid to download something from sources other than GooglePlay (because of frequent mobile one-click subscription scams)
Probably google's button could perform better.

**Comment by thusoy** [comment]
Hi! We haven't tested it yet, but I like the hypothesis so we might run some tests with it and see how it goes. I think we'll still have the current button since it's a bit clearer for those who already have the game, but I think including the device-appropriate app store link explicitly (which they would have been taken to anyway if they didn't have the app installed) might communicate a bit more clearly that that's where they'll end up. I'll keep you posted.

**Comment by Apollo144** [comment]
> Hi! We haven't tested it yet, but I like the hypothesis so we might run some tests with it and see how it goes. I think we'll still have the current button since it's a bit clearer for those who already have the game, but I think including the device-appropriate app store link explicitly (which they would have been taken to anyway if they didn't have the app installed) might communicate a bit more clearly that that's where they'll end up. I'll keep you posted.

Hey guys!
So will you test original Google&Apple download buttons?
I bet it's worth it.
https://imgurhd.ru/i/3zxc_orig.png
https://banner2.kisspng.com/20180330/rjw/kisspng-iphone-app-store-get-started-now-button-5abe1a35c6a840.4824190215224079898137.jpg

**Comment by thusoy** [comment]
Hi again! Yes, sorry for the long delay on this, we've been a bit swamped lately. We're planning to start experimenting with this quite soon, so you might start seeing this during the next weeks. We'll be rolling it out gradually to measure impact and see what is the best way to communicate this, so we'll probably try a couple options before we settle on something we roll out to everyone.

**Comment by Apollo144** [comment]
you're awesome guys! thx!

**Comment by thusoy** [comment]
The first test for this is being rolled out now, we'll be evaluating the results and iterating on it over the coming weeks. :)

**Comment by thusoy** [comment]
We aborted the first test of this since the app store buttons underperformed compared to the pink "Play now" button. This was the design we tested:
![tested design](https://user-images.githubusercontent.com/1466123/54570405-97613700-499b-11e9-967e-367d756fcef2.png)
If the app was available on both stores we showed badges for both stores side by side. This resulted in noticeably less engagement than the original. Let me know if there's any variation of this you think would be more engaging that we should try, we're very open to ideas. :)
